mrp_operations_extension/README.rst

```diff
@@ -0,0 +1,76 @@
+.. image:: https://img.shields.io/badge/licence-AGPL--3-blue.svg
+   :target: http://www.gnu.org/licenses/agpl-3.0-standalone.html
+   :alt: License: AGPL-3
+
+===============================
+MRP operations extension module
+===============================
+
+This module adds:
```

Comment:
@pedrobaeza module naming is generic (I never like blablable_extension modules) and I have the feeling that many different concepts are added in one module. 
Would  it be possible to split them in different more specific modules?